src/components/pages/PatientsSummary/functions.js

```diff
@@ -0,0 +1,25 @@
+import { get } from 'lodash';
+import { themeConfigs } from '../../../themes.config';
+
+/**
+ * This function returns initial panels number
+ * Default number is 4: Allergies, Medications, Contacts, Problems
+ *
+ * @return {number}
+ */
+export function getInitialPanelsNumber() {
+    const defaultPanelsNumber = 4;
+    const hiddenCorePlugins = get(themeConfigs, 'corePluginsToHide', []);
+    return defaultPanelsNumber - hiddenCorePlugins.length;
+}
+
+/**
+ *
+ * @param testStoreContent
+ * @return {*}
```

Comment:
@BogdanScherban this function should return a number, please correct the comment